scripts/mockdata/objects.json

```diff
@@ -14,8 +14,16 @@
       }
     ],
     "texts": ["", "hello", "from the other side"],
-    "is_interactable": false,
-    "animations_json": {}
+    "is_interactable": true,
+    "animations_json": {
+      "blackboardData": {
+        "componentType": "rotation-controls",
+        "jsonData": {
+          "position": [0, 0, 5],
+          "rotation": [0, 0, 0]
```

Comment:
Are these needed / are there times where we'd have different values here? Are we exposing this to admins?

Reply:
not sure. Rotation we might not need but position we may or may not have to keep. If certain objects are too large then when rotating them.. they will go into the blackboard or cover the camera. So positioning maybe be needed in that case